pkg/mimirtool/commands/testdata/apiserver.json

```diff
@@ -1470,7 +1470,7 @@
             "multi": false,
             "name": "cluster",
             "options": [ ],
-            "query": "label_values(apiserver_request_total, cluster)",
+            "query": "label_values(cluster)",
```

Comment:
without the corresponding regex change, existing tests would have failed because we'd extract "cluster" as a metric name as well.